src/main/java/org/eclipse/microprofile/starter/core/model/MicroProfileVersion.java

```diff
@@ -30,7 +32,7 @@
     // Order is important as it determines the order in the Dropdown on the screen.
     // @formatter:off
     NONE(null, "")
-    , MP30("3.0", "MP 3.0")
+    , MP30("3.0", "MP 3.0", Collections.singletonList("mp3_0"))
```

Comment:
List not required but for convenience (can do later on list.addAll()), maybe in future we have multiple alternative labels.

I used "mp_3.0" so that the directory the template is  has a clear description for what it is used.